### ISSUE TITLE
Issues/questions with OT literature 22.06 release

### ISSUE THREAD

**Issue by ravwojdyla** [opentargets/issues#2692]
I am looking at the release 22.06, and was doing some sanity checks/diffs between release 21.11 (previously used by us) and 22.06 and noticed some potential issues. Here I'm looking at the parquet matches. Would love to hear your feedback if these are issues or expected consequences of recent changes in the literature pipeline.

First here's high level comparison:

  | 21.11 | 22.06 | pct_change
-- | -- | -- | --
total rows | 313,715,701 | 418,759,081 | 0.33
total disease | 109,758,948 | 171,118,015 | 0.56
total genes | 145,979,721 | 176,446,133 | 0.21
total drugs | 57,977,032 | 71,194,933 | 0.23
unique PMID | 15,116,952 | 13,393,627 | (0.11)
unique disease | 12,819 | 10,760 | (0.16)
unique genes | 26,441 | 27,061 | 0.02
unique drugs | 9,821 | 9,539 | (0.03)

At high level the most important "issues" are:
 * roughly 11% decrease in the number of unique PMIDs with matches
 * roughly 16% decrease in the number of unique disease matched
 * there's 2,273,765 unique PMIDs that are in 21.11 but not in 22.06
 * there's 550,440 unique PMIDs that are in 22.06 but not in 21.11

Are these expected? And if there are, what was the source these changes?

<hr>

Some examples of specific PMIDs that are no longer available in 22.06, but that are present in 21.11:
 * [PMID:33562468](https://pubmed.ncbi.nlm.nih.gov/33562468/) has match to AREG, which is mentioned in the abstract
 * [PMID:33485463](https://pubmed.ncbi.nlm.nih.gov/33485463/) also mentioned mesothelioma in the title
   * this is a comment not an article (does this matter?)
 * [PMID:33369735](https://pubmed.ncbi.nlm.nih.gov/33369735/) mentioned THAP1 in the abstract

**Comment by ravwojdyla** [comment]
👋 @ireneisdoomed I'm curious if there's any updates on this? And specifically if this is on high level expected or potentially suspicious?

**Comment by ireneisdoomed** [comment]
Hi @ravwojdyla and sorry for the late response,

thank you very much for pointing this out. We have investigated the examples that you comment and in general we do see inconsistencies in the number of PubMed references analysed between pipeline runs. All the examples that you have provided are missing because they are not part of the submitted data, not because we are losing it somewhere downstream our pipelines.
In essence, there are no conceptual changes in how we process texts, indicating that there is a problem in the pipeline that does the NER on all eligible publications.

We have reported this to EPMC, so we hope we can get this solved soon. Thanks again for reporting.

_A breakdown of the examples provided:_

1. [PMID:33562468](https://pubmed.ncbi.nlm.nih.gov/33562468/)
- As reported, no matches appear to have been found for this article in 22.06 despite having data in 21.11.
- This PMID is not found in the raw EPMC submission on which the matches dataset is based (`gs://otar025-epmc/22.01/Abstracts`). Something was wrong in the EPMC pipeline that caused this ID not to be analysed.
- We have rerun our literature pipeline for our upcoming February release and this reference is picked up again.

2. [PMID:33485463](https://pubmed.ncbi.nlm.nih.gov/33485463/)
- As reported, no matches appear to have been found for this article in 22.06 despite having data in 21.11.
- Again, this is not found in the raw EPMC submission for the data in 22.06 and, in this case, neither in the new data that we will be ingesting.

Regarding the fact that the PMID refers to an article, this would have had to go through the pipeline anyway, as the only filters that are applied are the ones described in [the main publication](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5461726/pdf/13326_2017_Article_131.pdf):

```
1. Filter out all type of articles except “Research” articles (e.g. Reviews, Case Reports).
2. Filter out target-disease associations appearing in the following sections: Methods, References, Acknowledgement & Funding, Competing Interests, Author Contribution and Supplementary Material.
3. Filter out target-disease associations that appear only once in the body of a given article but not in the article's title or abstract.
```
These filters have not changed between 21.11 and 22.06.

3. [PMID:33369735](https://pubmed.ncbi.nlm.nih.gov/33369735/)
Same situation as case # 1.

**Comment by prashantuniyal02** [comment]
@DSuveges can we check this with the latest release?

**Comment by ravwojdyla** [comment]
I believe this was addressed in the 23.06, closing.
